Packs/FeedBlocklist_de/Integrations/FeedBlocklist_de/FeedBlocklist_de.py

```diff
@@ -0,0 +1,31 @@
+from CommonServerPython import *
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+
+    subfeeds = ['all', 'ssh', 'mail', 'apache', 'imap', 'ftp', 'sip', 'bots', 'strongips', 'ircbot', 'bruteforcelogin']
```

Comment:
ircbot is deprecated
```suggestion
    subfeeds = ['all', 'ssh', 'mail', 'apache', 'imap', 'ftp', 'sip', 'bots', 'strongips', 'bruteforcelogin']
```